CMakeLists.txt

```diff
@@ -500,6 +500,10 @@ if(EMSCRIPTEN)
   target_link_libraries(binaryen_wasm optimized "-Wno-error=closure")
   target_link_libraries(binaryen_wasm optimized "-flto")
   target_link_libraries(binaryen_wasm debug "--profiling")
+  # Avoid catching exit/rejection as that can confuse error reporting in Node,
+  # see https://github.com/emscripten-core/emscripten/issues/17228
+  target_link_libraries(binaryen_wasm "-sNODEJS_CATCH_EXIT=0")
+  target_link_libraries(binaryen_wasm "-sNODEJS_CATCH_REJECTION=0")
```

Comment:
`NODEJS_CATCH_REJECTION` is no longer needed here since it already defaults to 0 when targetting node > 15 (which is the default).

Reply:
Thanks, I verified it still works that way. Pushed.